Accessibility changes for the Change Log section (#201)
ubccr/ccrdocs: pages/changelogs/2025.md

 - Reboot of all cluster nodes
 - Updates of front-end login nodes (`login1`, `login2`, `vortex-future`)
 - Infrastructure Updates
 - Documentation updates to be published: Update to [CCR's software policy](../policies/software.md), [new information on the "R" Statistical package](../software/modules.md#r-the-statistical-computing-and-graphics-application) for software release ccrsoft/2024.04, additional information on using [QOS (Quality of Service) values](../hpc/jobs.md#slurm-directives-partitions-qos) for clarification, added a "change logs" section to our documentation site  which includes [downtime announcements](../changelogs/downtime-schedule.md).  All future announcements will be in the documentation.
 - VSCode OnDemand app form changes - removed field to load additional software modules when starting.  This doesn't propagate to the VSCode environment properly so it is being removed.  Documentation is updated to reflect this change.
 - VMD OnDemand app form changes - added fields to allow users to request memory, CPUs, and GPUs rather than capping jobs at 64GB of RAM.  

>  - Release of new "Using Python at CCR" course in UB Learns.  Self-register on the [UB Learns websit websitee](https://ublearns.buffalo.edu/d2l/le/discovery/view/course/288741)  

Reviewer: ```suggestion
 - Release of new "Using Python at CCR" course in UB Learns.  Self-register on the [UB Learns website](https://ublearns.buffalo.edu/d2l/le/discovery/view/course/288741)  
```

Users may test this out using the command: `module load ccrsoft/2024.04` 
You can use the `vortex-future` login node where this environment is now the default: `vortex-future.ccr.buffalo.edu`   
The default software environment on the login (`vortex.ccr.buffalo.edu`), compile, and compute nodes, as well as in OnDemand, remains `ccrsoft/2023.01` at this time.  
You can pin your account to a specific release follow the "Pin to a specific release" section of [CCR's Software Environment Documentation](../software/releases.md)

> in the green box on [this page](https://docs.ccr.buffalo.edu/en/latest/software/releases/) 

Reviewer: ```suggestion
```
I think you meant to delete this

You can see the node hardware specs with the following commands: 
`snodes all ub-hpc/scavenger |grep CPU-Platinum-8562Y`

For instructions on how to use the scavenger partitions, please refer to [CCR's job scavenger documentation](../hpc/jobs.md#scavenging-idle-cycles). 

For a full list of specs for the equipment in the UB-HPC academic partitions, see [CCR's "ub-hpc" cluster hardware documentation](https://www.buffalo.edu/ccr/support/research_facilities/ub-hpc/academic-hardware-specs.html).

> For the industry partitions (and ub-hpc scavenger partition), see [CCR's "faculty" cluster hardware documentation](https://www.buffalo.edu/ccr/support/research_facilities/industry_cluster.html).

Reviewer: ```suggestion
For the industry partitions (and ub-hpc scavenger partition), see [CCR's "ub-hpc" cluster hardware documentation](https://www.buffalo.edu/ccr/support/research_facilities/industry_cluster.html).
```
The industry partitions are in the ub-hpc cluster
